mapdata/WarcraftLegacies/Doodads/1024_10240.json

```diff
@@ -295,7 +295,7 @@
       "Y": 10176,
       "Z": 281.5
     },
-    "Rotation": -1.4017224,
+    "Rotation": 4.881463,
```

Comment:
Not sure what happened here but can you revert the changes in the Doodads and Units folders?